### ISSUE TITLE
Random Error

### ISSUE THREAD

**Issue by Alucard2112** [Ellypse/Storyline#45]
The addon still functions but this error-message is displayed:
Date: 2019-10-18 12:42:11
ID: 1
Error occured in: Global
Count: 1
Message: ..\AddOns\Storyline\events.lua line 278:
   attempt to index global 'QuestNPCModel' (a nil value)
Debug:
   Storyline\events.lua:278:
      Storyline\events.lua:263
   Storyline\events.lua:487:
      Storyline\events.lua:466
   Storyline\events.lua:549:
      Storyline\events.lua:497
   Storyline\events.lua:571:
      Storyline\events.lua:555
   Storyline\logic.lua:286:
      Storyline\logic.lua:193
   Storyline\events.lua:866: callback()
   ...ddOns\Storyline\lib\Ellyb\Tools\EventsDispatcher.lua:74: TriggerEvent()
   ...face\AddOns\Storyline\lib\Ellyb\Tools\GameEvents.lua:43:
      ...face\AddOns\Storyline\lib\Ellyb\Tools\GameEvents.lua:41
Locals:
isOnCompleteStep = false
questPortrait = 79674
questPortraitText = "Something dark and disturbing lies in wait. It must be destroyed."
questPortraitName = "The Void Horror"
questPortraitMount = 0
(*temporary) = nil
(*temporary) = nil
(*temporary) = 79674
(*temporary) = 0
(*temporary) = "Something dark and disturbing lies in wait. It must be destroyed."
(*temporary) = "The Void Horror"
(*temporary) = -16
(*temporary) = -48
(*temporary) = "attempt to index global 'QuestNPCModel' (a nil value)"
Storyline_NPCFrame = Storyline_NPCFrame {
 0 = <userdata>
 TopBorder = <unnamed> {
 }
 debug = Storyline_NPCFrameDebug {
 }
 BorderBottomLeft = <unnamed> {
 }
 BorderTopRight = <unnamed> {
 }
 BorderLeft = <unnamed> {
 }
 banner = Storyline_NPCFrameBanner {
 }
 chat = Storyline_NPCFrameChat {
 }
 models = Storyline_NPCFrameModels {
 }
 title = Storyline_NPCFrameTitle {
 }
 dynamicBackground = <unnamed> {
 }
 BottomBorder = <unnamed> {
 }
 BorderRight = <unnamed> {
 }
 BorderBottomRight = <unnamed> {
 }
 BorderTopLeft = <unnamed> {
 }
}

AddOns:
  KalielsTracker, v3.1.10
  Swatter, v8.2.6377 (SwimmingSeadragon)
  AngryWorldQuests, v
  AtlasLoot, vv8.10.00
  AtlasLootBattleforAzeroth, vv8.10.00
  AtlasLootBurningCrusade, vv8.10.00
  AtlasLootCataclysm, vv8.10.00
  AtlasLootClassic, vv8.10.00
  AtlasLootLegion, vv8.10.00
  AtlasLootMistsofPandaria, vv8.10.00
  AtlasLootWarlordsofDraenor, vv8.10.00
  AtlasLootWrathoftheLichKing, vv8.10.00
  AutoJunkSeller, vv8.2.0
  BagBrother, v
  Bagnon, v8.2.16
  Bartender4, v4.8.8
  BeanCounter, v8.2.6422 (SwimmingSeadragon)
  BetterPowerBarAlt, v1.5.3
  BFAInvasionTimer, vv8.2.5
  DBMAzerothBfA, v
  DBMCore, v8.2.24
  DBMDefaultSkin, v
  DBMStatusBarTimers, v
  EasyScrap, v24
  ExRT, v4060
  HandyNotes, vv1.5.9
  HandyNotesBattleForAzerothTreasures, v1
  HandyNotesLegionRaresTreasures, v1.1
  HandyNotesLongForgottenHippogryph, v1
  HandyNotesPandariaTreasures, v1.06
  HandyNotesSuramarLocations, v0.0.1
  HandyNotesTreasureHunter, v1
  HandyNotesWarfrontRares, v0.5.0
  kgPanels, v1.24
  Mapster, v1.8.5
  Masque, v8.2.7
  MasqueDaedUI, v1.0
  MoveTalkingHead, vv1.3.2-1-g0d2672c
  Narcissus, v1.0.5b
  NigglesPetTeams, v3.05
  ParagonReputation, v1.24
  PitBull4, vv4.1.21
  Postal, vv3.6.0
  Prat30, v3.7.36
  Prat30Libraries, v
  Quartz, v3.5.2
  Rarity, v1.0 (r692-release)
  RarityOptions, v
  SexyMap, vv8.2.2
  Skada, v1.7.6
  SlideBar, v8.2.6375 (SwimmingSeadragon)
  Storyline, v3.1.2
  Stubby, v8.2.6376 (SwimmingSeadragon)
  SuperDuperMacro, v8.0.1.1
  TomTom, vv80205-1.0.7
  XLoot, v8.1-1
  XLootFrame, v8.1-1
  XLootGroup, v8.1-1
  XLootMaster, v8.1-1
  XLootMonitor, v8.1-1
  XPMultiBar, v8.2.2-final
  BlizRuntimeLib_enUS v8.2.5.80205 <none>
  (ck=681)

**Comment by sarcoex** [comment]
I get the same error sometimes. And the 3D model for the NPC I talk to is sometimes not shown, most of the time the 3D model is shown, even if the error occur. It does not happen on every quest though.
My recent error happened when talking to Hobart Grapplehammer at The Lost Isles, at Volcanoth's Lair. It happened during the "intro" dialog before accepting the quest "Volcanoth!".
With only Storyline, BugGrabber and Bugsack activated, the error still occurs.

Here is my error:
```
33x Storyline\events.lua:278: attempt to index global 'QuestNPCModel' (a nil value)
Storyline\events.lua:278: in function <Storyline\events.lua:263>
Storyline\events.lua:487: in function <Storyline\events.lua:466>
Storyline\events.lua:549: in function <Storyline\events.lua:497>
Storyline\events.lua:571: in function <Storyline\events.lua:555>
Storyline\logic.lua:286: in function <Storyline\logic.lua:193>
Storyline\events.lua:866: in function `callback'
...ddOns\Storyline\lib\Ellyb\Tools\EventsDispatcher.lua:74: in function `TriggerEvent'
...faceStoryline\lib\Ellyb\Tools\GameEvents.lua:43: in function <...faceStoryline\lib\Ellyb\Tools\GameEvents.lua:41>

Locals:
isOnCompleteStep = false
questPortrait = 33511
questPortraitText = "Use Harris's Ampule to tranquilize him."
questPortraitName = "Lurking Worgen"
questPortraitMount = 0
(*temporary) = nil
(*temporary) = nil
(*temporary) = 33511
(*temporary) = 0
(*temporary) = "Use Harris's Ampule to tranquilize him."
(*temporary) = "Lurking Worgen"
(*temporary) = -16
(*temporary) = -48
(*temporary) = "attempt to index global 'QuestNPCModel' (a nil value)"
Storyline_NPCFrame = Storyline_NPCFrame {
 0 = <userdata>
 BorderBottomLeft = <unnamed> {
 }
 banner = Storyline_NPCFrameBanner {
 }
 chat = Storyline_NPCFrameChat {
 }
 LocBack = <unnamed> {
 }
 SetSize = <function> defined =[C]:-1
 dynamicBackground = <unnamed> {
 }
 debug = Storyline_NPCFrameDebug {
 }
 BorderTopLeft = <unnamed> {
 }
 BorderTopRight = <unnamed> {
 }
 BottomBorder = <unnamed> {
 }
 isSizing = false
 BorderRight = <unnamed> {
 }
 models = Storyline_NPCFrameModels {
 }
 LocFore = <unnamed> {
 }
 title = Storyline_NPCFrameTitle {
 }
 fadeInfo = <table> {
 }
 TopBorder = <unnamed> {
 }
 BorderLeft = <unnamed> {
 }
 BorderBottomRight = <unnamed> {
 }
 LocMid = <unnamed> {
 }
}
```
![image](https://user-images.githubusercontent.com/1639539/67156446-e18a0c80-f31e-11e9-80da-e7a883c15b8d.png)
![image](https://user-images.githubusercontent.com/1639539/67156463-1e560380-f31f-11e9-9eaf-3c1a2a7ce4f6.png)

Addons:
Auctionator
Azeroth Adventure Album
Barrels o' Easy
Blizzmove
BugGrabber
BugSack
Can I Mog It?
CollectionShop
Deadly Boss Mods
Grid2
Kaliel's Tracker
Kui Nameplates
Minimap Button Frame
Narcissus
OmniCC
OneClickEnchantScroll
Quest Completist
Recount
Scrap
Sexymap
Storyline
Tetra

**Comment by Ellypse** [comment]
This issue is now fixed in version 3.1.3.
